app/styles/player.styl

```diff
@@ -226,20 +226,35 @@
             display: none
 
         #volume-bar
+            position: absolute
             width: 90px
+            height: 100%
+            border-radius: 2px
+            display: inline-block
+            cursor: pointer
+
+        #volume-bar::before,
+        #volume
+            position: absolute
+            content: ""
+            width: 100%
             height: 4px
             border-radius: 2px
+            top: 34px
             background-color: $border-color
-            display: inline-block
+
+        #volume-bar::-moz-before,
```

Comment:
This ruleset block is useless, as `-moz-before` pseudo-element is deprecated since a long time. We can remove the whole block safely